src/main/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayout.java

```diff
@@ -122,22 +122,21 @@ private void appendStackTrace(StringBuilder log, ThrowableProxy proxy) {
         }
     }
 
-    private void loopCauses(StringBuilder log, ThrowableProxy parentProxy, int depth) {
-        ThrowableProxy cause = (ThrowableProxy) parentProxy.getCause();
+    private void logCause(StringBuilder log, ThrowableProxy cause, int depth) {
```

Comment:
can we just bin this class and go back to using logback pattern formatter?
i.e. do we actually need it?

Reply:
don't see the use. don't see the point. don't see. this was wasteful time of development on some requirement some1 invented.

but should be logged and planned about

Reply:
I'm a great fan of pattern formatter, as it's mature, clear and works great. We've got these error codes and other stuff here, and I don't want to break things I don't understand. But it's worth reviewing this and maybe replacing with pattern formatter. Might be a proper story.

Reply:
i could search for that story, but yea, again, this was just a wasteful time to implement some bad idea invented by some architect or whoever alike

Reply:
I would be wholely in favour of binning this
In cmc we stopped using it and used our own appender because this one was 💩

Reply:
I've created this story to address the comments: https://tools.hmcts.net/jira/browse/RPE-839 . Please feel free to edit it. For the time being, though, I think it's better to have this fix than not.

Reply:
> i could search for that story, but yea, again, this was just a wasteful time to implement some bad idea invented by some architect or whoever alike

True, there were some weird programme-wide requirements for logging.